components/IscnRegisterForm.vue

```diff
@@ -135,9 +163,9 @@
             >{{ $t('IscnRegisterForm.label.numbersProtocol.details.link') }}</Link>
           </i18n>
         </CheckBox>
-      </FormField>
+      </FormField> -->
```

Comment:
Please keep this checkbox, tick means upload all files to number, none means not